src/main/scala/dpla/ingestion3/utils/Utils.scala

```diff
@@ -100,7 +98,8 @@ object Utils {
     * @param runtime Runtime in milliseconds
     * @param recordCount Number of records output
     */
-  def printResults(runtime: Long, recordCount: Long): Unit = {
+  def logResults(runtime: Long, recordCount: Long, logger: Logger): Unit = {
+    // TODO figure out a better way to share a logger...
```

Comment:
We probably don't want to share loggers, but rather configure the appender and make sure different loggers use the same appender. But even then, when we're in distributed mode, that won't be possible across all the executors.

Reply:
The other option I was considering last night was using System.properties to store what we need to dynamically name output files and I think that would allow us to just load the `ingestion3` logger from multiple classes and get the output routed correctly.

Reply:
In general, they want you to use one Logger per class and use the Appender to talk about where things go. 

This is going to get complicated though when/if we use multiple machines to do this work, it's not going to magically go over the wire back to the master system. And I'm not sure stuff in System properties is distributed either. 

If we really wanted to avoid logging to Spark's logs, probably we would need the information to instantiate an Appender to be serialized over the wire, and the Loggers to be created from that. And then at job wrap up, we'd need something to go collect the logs.

In the long run, it might actually be easier to just piggyback on Spark's Appender and deal with the merged streams, because we'd be reinventing this wheel in a distributed context (YARN and EMR do log aggregation and archiving for you.)

But right now, we're not distributed, so let's just keep this.